x/evm/keeper/msg_server.go

```diff
@@ -784,10 +792,12 @@ func (s msgServer) VoteConfirmDeposit(c context.Context, req *types.VoteConfirmD
 	event := sdk.NewEvent(types.EventTypeDepositConfirmation,
 		sdk.NewAttribute(sdk.AttributeKeyModule, types.ModuleName),
 		sdk.NewAttribute(types.AttributeKeyChain, chain.Name),
+		sdk.NewAttribute(types.AttributeKeySourceChain, chain.Name),
```

Comment:
Added this for consistency. In the next upgrade, we can remove the old key and update vald too.